app/scripts/lib/xhr.js

```diff
@@ -67,6 +67,7 @@ define([
         // make sure to set the dataType for Firefox <21. See issue #1930
         dataType: 'json',
         type: options.type,
+        timeout: options.timeout || 5000,
```

Comment:
@shane-tomlinson Or I could just set it to `|| 0` instead of or `|| 5000` seeing as thats the jquery default if no timeout is set, would that be all right?